### PR TITLE
RtpSender and track

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/w3c/webrtc-pc/issues/1064

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -4360,9 +4360,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
     is managed through objects called <code><a>RTCRtpSender</a></code>s.
     Similarly, the reception and decoding of <code>MediaStreamTrack</code>s is
     managed through objects called <code><a>RTCRtpReceiver</a></code>s. Each
-    track to be sent is associated with exactly one
-    <code><a>RTCRtpSender</a></code>, and each track to be received is
-    associated with exactly one <code><a>RTCRtpReceiver</a></code>.</p>
+    <code><a>RTCRtpSender</a></code> is associated with exactly one track,
```
Comment (taylor-b) [inline webrtc.html]: "zero or one", or "at most one"?

Comment (aboba) [inline webrtc.html]: Fixed.

**Comment by alvestrand** [comment]
lgtm

**Comment by stefhak** [comment]
lgtm

**Pull request merged**